model-providers/mistral/runtime/src/main/java/io/quarkiverse/langchain4j/mistralai/QuarkusMistralAiClient.java

```diff
@@ -116,6 +118,12 @@ public MistralAiEmbeddingResponse embedding(MistralAiEmbeddingRequest request) {
         return restApi.embedding(request, apiKey);
     }
 
+    // TODO: we don't provide support for MistralAiModerationModel yet
```

Comment:
We don't?

Reply:
Well, users can instantiate it directly, but we don't have a declarative way via config properties for it yet. I plan to add that via https://github.com/quarkiverse/quarkus-langchain4j/issues/1095

Reply:
Gotcha, thanks